lib/lte_link_control/modules/Kconfig

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config LTE_LC_CONEVAL
+	bool "Enable evaluation of connection parameters"
+	default n
```

Comment:
```suggestion
```

from https://docs.zephyrproject.org/latest/build/kconfig/tips.html#redundant-defaults
> `bool` symbols implicitly default to `n`, and `string` symbols implicitly default to the empty string.
> Therefore, `default n` and `default ""` are (almost) always redundant.

Same comment applies on Kconfig settings below in this file.